src/applications/disability-benefits/all-claims/pages/recentJobApplications.js

```diff
@@ -0,0 +1,109 @@
+import { merge, omit } from 'lodash';
+import fullSchema from '../config/schema';
+
+const { date } = fullSchema.definitions;
+
+import {
+  recentJobApplicationsDescription,
+  substantiallyGainfulEmployment,
+} from '../content/recentJobApplications';
+
+import {
+  uiSchema as addressUI,
+  schema as addressSchema,
+} from '../../../../platform/forms/definitions/address';
+
+const address = addressSchema(fullSchema);
+
+import { validateZIP } from '../validations';
+
+import currentOrPastDateUI from 'us-forms-system/lib/js/definitions/date';
+import RecentJobApplicationField from '../components/RecentJobApplicationField';
+
+export const uiSchema = {
+  'ui:title': 'Recent job applications',
+  'view:hasAppliedEmployers': {
+    'ui:title': recentJobApplicationsDescription(),
+    'ui:widget': 'yesNo',
+  },
+  appliedEmployers: {
+    'ui:options': {
+      viewField: RecentJobApplicationField,
+      expandUnder: 'view:hasAppliedEmployers',
+      itemName: 'Job',
+    },
+    items: {
+      name: {
+        'ui:title': "Company's Name",
+      },
+      address: merge(addressUI('', false), {
+        'ui:order': [
+          'country',
+          'addressLine1',
+          'addressLine2',
+          'city',
+          'state',
+          'zipCode',
+        ],
+        addressLine1: {
+          'ui:title': 'Street 1',
+        },
+        addressLine2: {
+          'ui:title': 'Street 2',
+        },
+        state: {
+          'ui:title': 'State',
+        },
+        zipCode: {
+          'ui:title': 'Postal Code',
+          'ui:validations': [validateZIP],
+        },
+      }),
+      workType: {
+        'ui:title': 'Type of work',
+      },
+      date: currentOrPastDateUI('Date you applied'),
+    },
+  },
+  substantiallyGainfulEmploymentInfo: {
```

Comment:
can we change this to `'view: substantiallyGainfulEmploymentInfo'`. Or something with `view:` in it. We use that to symbolize UI related data fields/content fields(things not in the form Schema).